src/models/transfers.js

```diff
@@ -318,6 +318,12 @@ function * fulfillTransfer (transferId, fulfillmentUri) {
   const fulfillment = ConditionFulfillment.fromDataExternal(fulfillmentUri)
   fulfillment.setTransferId(transferId)
   const existingFulfillment = yield db.transaction(function * (transaction) {
+    // Set isolation level to avoid reading "prepared" transaction that is currently being
+    // executed by another request. This ensures the transfer can be fulfilled only once.
+    if (db.client === 'pg') {
```

Comment:
I don't see it either ... I guess it wasn't saved properly. Long story short, I can't set the isolation level for mysql. I get `ERROR Error: ER_CANT_CHANGE_TX_CHARACTERISTICS: Transaction isolation level can't be changed while a transaction is in progress
`. It looks like it has to be set once on start up, rather than on a transaction by transaction basis.

I also can't set oracle, because we are using XE, which doesn't support this. Once we move to the standard edition, we should be able to set the isolation level on oracle.

I think this is OK for this ticket, since this is a blocker for RC, and has only been observed on postgres. I'm all for a most robust solution long term, but I don't want to delay this until later in the week, just to wait for Oracle standard edition.